code-of-conduct-contact.md

```diff
@@ -0,0 +1,73 @@
+
+We do not look negatively on you if we find the incident is not a violation.
+
+And knowing about incidents that are not violations, or happen outside our spaces, can also help us to improve the Code of Conduct or the processes surrounding it.
+
+## In your report please include:
+
+- Your contact info _(so we can get in touch with you if we need to follow up)_
+- Names _(real, nicknames, or pseudonyms)_ of any individuals involved. If there were other witnesses besides you, please try to include them as well.
+- When and where the incident occurred. Please be as specific as possible.
+- Your account of what occurred. If there is a publicly available record (e.g. a mailing list archive or a public IRC logger) please include a link and/or screenshot.
+- Any extra context you believe existed for the incident.
+- If you believe this incident is ongoing.
+- Any other information you believe we should have.
+
+
+## What happens after you file a report?
+
+You will receive an email from the dat consortium, acknowledging receipt immediately.
+We promise to acknowledge receipt within 24 hours _(and will aim for much quicker than that)_.
+
+The working group will immediately meet to review the incident and determine:
+
+- What happened.
+- Whether this event constitutes a code of conduct violation.
+- Who the bad actor was.
+- Whether this is an ongoing situation, or if there is a threat to anyone's physical safety.
+- If this is determined to be an ongoing incident or a threat to physical safety, the working groups' immediate priority will be to protect everyone involved.
+    This means we may delay an "official" response until we believe that the situation has ended and that everyone is physically safe.
+
+Once the working group has a complete account of the events they will make a decision as to how to response. Responses may include:
+
+- Nothing _(if we determine no violation occurred)_.
+- A private reprimand from the working group to the individual(s) involved.
+- A public reprimand.
+- An imposed vacation _(i.e. asking someone to "take a week off" from a mailing list or IRC)_.
+- A permanent or temporary ban from some or all Dat Ecosystem spaces _(Discord Chat, mailing lists, IRC, etc.)_.
+- A request for a public or private apology.
+
+We'll respond within one week to the person who filed the report with either a resolution or an explanation of why the situation is not yet resolved.
+
+Once we've determined our final action, we'll contact the original reporter to let them know what action _(if any)_ we'll be taking.
+
+We'll take into account feedback from the reporter on the appropriateness of our response, but we don't guarantee we'll act on it.
+
+The consortium members may choose to a public report of the incident.
+
+## What if your report concerns a possible violation by a consortium member?
+
+If your report concerns a current member of the consortium, you may not feel comfortable sending your report to the committee,
```

Comment:
committee? or working group?

Reply:
In my thinking: You send it to the working group's email but the entire consortium (committee?) will be able to see it.

Reply:
Maybe comittee sounds to rigid and working group could be used as a synonym. That's at least how I understood the comment.